.github/workflows/release.yml

```
@@ -0,0 +1,121 @@
+        echo "${{ secrets.PROD_DOCKER_COMPOSE }}" | base64 --decode > deploy/docker-compose.yml
+        
+    - name: Make Script Executable
+      run: chmod +x deploy/deploy.sh
+        
+    - name: Deploy
+      run: ./deploy/deploy.sh
+      
+    - name: Cleanup
+      run: |
+        docker system prune -af --volumes
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)